rudder-server-relay/SOURCES/rudder-apache-relay-ssl.conf

```diff
@@ -34,3 +34,35 @@ RewriteRule /policies/(.*?)/(.*) /var/rudder/share/$1/$2
     Include /opt/rudder/etc/rudder-networks-24.conf
   </IfVersion>  
 </Directory>
+
+
+<Location /rudder/relay-api/shared-folder>
+
+  <IfVersion < 2.4>
+    Order allow,deny
+    Include /opt/rudder/etc/rudder-networks.conf
+  </IfVersion>
+  <IfVersion >= 2.4>
+    Include /opt/rudder/etc/rudder-networks-24.conf
+  </IfVersion>
+
+</Location>
+
+RewriteCond %{REQUEST_METHOD}  =GET
+RewriteRule /rudder/relay-api/shared-folder/(.*) /var/rudder/configuration-repository/shared-files/$1
+
+<Directory /var/rudder/configuration-repository/shared-files>
```

Comment:
No, both are checked by apache.
It checks Location when hitting this location. Then, aliases and rewrites are done, then if is matches a path on the server, it checks Directory and then it check File.
So in this case it will hit both location and Directory which is redundant